feature-libs/product/configurators/rulebased/cpq/cpq-configurator-normalizer.ts

```diff
@@ -0,0 +1,195 @@
+import { Injectable } from '@angular/core';
```

Comment:
Test coverage is too low, see coverage report

Reply:
done